packages/ui/v2/core/Shell.tsx

```diff
@@ -170,7 +170,7 @@ type LayoutProps = {
   TopNavContainer?: ReactNode;
   drawerState?: DrawerState;
   HeadingLeftIcon?: ReactNode;
-  backPath?: string; // renders back button to specified path
+  backPath?: string | boolean; // renders back button to specified path
```

Comment:
backPath will either navigate to the route if it is a string or just do router.back.